[DE] Update `Medals` and `Medals/Unlock requirements/Dedication` (#12784)
ppy/osu-wiki: wiki/Medals/Unlock_requirements/Dedication/de.md

| 400.000 Keys | Treffe insgesamt 400.000 Tasten in osu!mania. | Four hundred thousand and still not even close. |
| 4.000.000 Keys | Treffe insgesamt 4.000.000 Tasten in osu!mania. | Is this the end of the rainbow? |
| 40.000.000 Keys | Treffe insgesamt 40.000.000 Tasten in osu!mania. | When someone asks which keys you play, the answer is now 'yes'. |

## Tägliche Herausforderung

| Medaillenname | Anforderung | Beschreibung |
| :-- | :-- | :-- |

> | Daily Sprout | Erreiche eine 1-tägige Serie in [Tägliche Herausforderung](/wiki/Gameplay/Daily_challenge). | Ready for anything. |

Reviewer: I don't think I've heard a streak be translated as "Serie" before.. Is there a precedent for this? I would've imagined the translation would need to be less literal by doing something akin to `Gewinne/Bestehe die [taegliche Herausforderung](...) sieben Mal hintereinander`

Author: This sounds even better 👍 

```suggestion
| Daily Sprout | Gewinne die [tägliche Herausforderung](/wiki/Gameplay/Daily_challenge) einmal. | Ready for anything. |
```

> | Weekly Sapling | Erreiche eine 7-tägige Serie in [Tägliche Herausforderung](/wiki/Gameplay/Daily_challenge). | Circadian rhythm calibrated. |

Author: ```suggestion
| Weekly Sapling | Gewinne die [tägliche Herausforderung](/wiki/Gameplay/Daily_challenge) 7 Mal hintereinander. | Circadian rhythm calibrated. |
```

> | Monthly Shrub | Erreiche eine 30-tägige Serie in [Tägliche Herausforderung](/wiki/Gameplay/Daily_challenge). | In for the grind. |

Author: ```suggestion
| Monthly Shrub | Gewinne die [tägliche Herausforderung](/wiki/Gameplay/Daily_challenge) 30 Mal hintereinander. | In for the grind. |
```
